packages/next/src/client/components/react-dev-overlay/_experimental/internal/container/RuntimeError/component-stack-pseudo-html.tsx

```diff
@@ -295,41 +304,105 @@ export function PseudoHtmlDiff({
 
   return (
     <div data-nextjs-container-errors-pseudo-html>
-      <button
-        tabIndex={10} // match CallStackFrame
-        data-nextjs-container-errors-pseudo-html-collapse
-        onClick={() => toggleCollapseHtml(!isHtmlCollapsed)}
-      >
-        <CollapseIcon collapsed={isHtmlCollapsed} />
-      </button>
+      <LineDiv>
+        <button
+          tabIndex={10} // match CallStackFrame
+          data-nextjs-container-errors-pseudo-html-collapse
+          onClick={() => toggleCollapseHtml(!isHtmlCollapsed)}
+        >
+          <CollapseIcon collapsed={isHtmlCollapsed} />
+        </button>
+      </LineDiv>
       <pre {...props}>
         <code>{htmlComponents}</code>
       </pre>
     </div>
   )
 }
 
+function LineDiv({
```

Comment:
I was using `span` because there're a lot of places have natural line break `\n` ending. So most of places can still be span. Or you need to change a lot of places of text modification of `\n`. The line break in the text should still be able to show the break.

I feel we can keep `span` for less modification to avoid introducing new issues. And the only place you need to have a block or div line is the place above where you wrap the `button`.

```suggestion
function Line({
```